R/datatypes-s3-base.R

```diff
@@ -1,3 +1,37 @@
+#' @export
+`$.polars_dtype_enum` <- function(x, name) {
+  # Enum only method `union`
```

Comment:
Can you add a comment explaining this is why it's not in `generate-S3methods-dollar.R`?